src/om_bootstrap/nav.cljs

```diff
@@ -171,8 +179,9 @@
 (defn render-navbar-child [owner child bs]
   (let [f (fn [props]
             (let [opts (:opts props)
-                  collapsible? (when (:toggle-nav-key bs)
-                                 (= (:key opts) (:toggle-nav-key bs)))
+                  collapsible? (or (:collapsible? opts)
```

Comment:
We have to check for :collapsible? here, otherwise if you add :collapsible? true to your nav props, it'll be overwritten as false if you don't supply the :toggle-nav-key in navbar, and the corresponding :key in the nav props